src/20_Components/amp-bind.html

```diff
@@ -101,27 +101,29 @@
    set when a state change is triggered (see below).
 
 -->
-<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+<div class="p1">
+  <p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+</div>
 
 <!-- ## Binding Styles -->
 <!--
   Styling can be changed by applying css classes. It will override all the element's style classes. `allAnimals` is the `id` of the &lt;amp-state&gt; component above.
 -->
-
-<p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
-
+<div class="p1">
```

Comment:
didn't notice - sorry!

I'm wondering whether it'd make sense to add a default margin for paragraphs in the preview, e.g.:

```
.abe-preview > p {
  margin: 1rem;
}

What do you think?